packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -195,6 +196,73 @@ export const globalOptions = {
       'Version of the kadena/devnet Docker image to use (e.g. "latest")',
     ),
   }),
+  //Simulation
+  numberOfAccounts: createOption({
+    key: 'accounts' as const,
+    prompt: simulationPrompts.numberOfAccountsPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-a, --accounts <accounts>',
+      'Amount of accounts to be created in the simulation.',
+    ),
+  }),
+  transferInterval: createOption({
+    key: 'interval' as const,
+    prompt: simulationPrompts.transferIntervalPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-i, --interval <interval>',
+      'Transfer interval in milliseconds.',
+    ).argParser((value) => parseInt(value, 10)),
+  }),
+
+  maxTransferAmount: createOption({
+    key: 'maxTransferAmount' as const,
+    prompt: simulationPrompts.maxTransferAmountPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-x, --max-transfer-amount <maxTransferAmount>',
+      'Max transfer amount per single transaction (coin).',
+    ),
+  }),
+
+  tokenPool: createOption({
+    key: 'tokenPool' as const,
+    prompt: simulationPrompts.tokenPoolPrompt,
+    validation: z.number(),
+    option: new Option(
+      '-p, --token-pool <tokenPool>',
+      'Total token pool (coin).',
+    ),
+  }),
+  seed: createOption({
+    key: 'seed' as const,
+    prompt: simulationPrompts.seedPrompt,
+    validation: z.string(),
+    option: new Option('-s, --seed <seed>', 'Seed for the simulation.'),
+  }),
+  logFolder: createOption({
+    key: 'logFolder' as const,
+    prompt: generic.logFolderPrompt,
+    validation: z.string(),
+    option: new Option(
+      '-l, --log-folder <logFolder>',
+      'Directory where the log file will be generated. (e.g. ./kadena/simulation-logs/',
+    ),
+  }),
+  defaultChainId: createOption({
+    key: 'defaultChainId' as const,
+    prompt: simulationPrompts.defaultChainIdPrompt,
+    validation: z.string(),
+    transform: (chainId: string) => {
+      return chainId as ChainId;
+    },
+    option: new Option(
+      '-c, --default-chain-id <defaultChainId>',
+      'Default chain id for the simulation (eg. 0).',
+    ),
+  }),
```

Comment:
Maybe we can move the options specifically for simulation to it's own options set.